bin/luigi

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env python
 
-import luigi
+import sys
+import warnings
+import luigi.cmdline
+
+
+def main(argv):
+    warnings.warn("'bin/luigi' has moved to console script 'luigi'", DeprecationWarning)
```

Comment:
Ah, I've always done tricks with `PYTHONPATH`, maybe I should install luigi the "proper" way. :)